vlib/v/checker/tests/return_fixed_array.out

```diff
@@ -1,7 +1,15 @@
-vlib/v/checker/tests/return_fixed_array.vv:11:25: error: fixed array cannot be returned by function using alias
+builder error: redefinition of function `return_fixed_array`
+vlib/v/checker/tests/return_fixed_array.vv:3:1: conflicting declaration: fn return_fixed_array() [3]int
+    1 | type Abc = [3]int
+    2 | 
+    3 | fn return_fixed_array() [3]int {
+      | ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    4 |     return [1, 2, 3]!
+    5 | }
```

Comment:
imho just delete the whole function in `vlib/v/checker/tests/return_fixed_array.vv` .

Reply:
Done.